lib/generators/rodauth/templates/app/views/rodauth/tailwind/close_account.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title, rodauth.close_account_page_title %>
+<div class="dark:bg-gray-900 flex items-center justify-center h-screen">
+  <div class="mx-4 py-4">
+    <h3 class="mb-3 text-3xl font-semibold text-center text-white"> Close Account </h3>
```

Comment:
I noticed some form titles are `<h2>` and some `<h3>`. Would it make sense to use the same heading for all forms? `<h2>` would seem reasonable to me.

Reply:
Fixed with: [f0a5307](https://github.com/janko/rodauth-rails/pull/114/commits/f0a5307420273872e47bf8bc9e76e3b1cffd4891)